starter-project-web-vue2/store/aymen.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios'
```

Comment:
no need to import axios. `this.$axios` is a global variable which will injected by nuxt